src/app/base/services/record/record-apollo.service.ts

```diff
@@ -533,13 +533,26 @@ export class RecordApolloService {
   }
 
   //private
-  #tokenMapper(token) {
+  #tokenMapper(token: any, tokensLength: number) {
+    let countLineBreaks = 0;
+    if (token.value.includes("\n")) {
+      countLineBreaks = token.value.split("\n").length - 1;
+    }
+    if (countLineBreaks > 0) {
+      // If we are on the first or last token and either/both of them is new lines, the class w-full cannot work because we don't have a previous or next token, that's why we need the original countLineBreaks
+      // If we are not on the first or last token, the array of the countLineBreaks has to be one less than the actual countLineBreaks because we use the class w-full of the current line as one line break
+      // Adding a completely new line and having a text that needs a new line are different in terms of css classes
+      const checkIfOrLastIdx = token.idx == 0 || token.idx == tokensLength;
+      countLineBreaks = checkIfOrLastIdx ? countLineBreaks : countLineBreaks - 1;
+    }
     return {
       value: token.value,
       idx: token.idx,
       posStart: token.posStart,
       posEnd: token.posEnd,
       type: token.type,
+      countLineBreaks: countLineBreaks,
+      countLineBreaksArray: countLineBreaks != 0 ? Array(countLineBreaks - 1).fill(0) : null,
```

Comment:
why do we need this?
looks to me like ist an array with 0 (or null)
we can also easily just loop between 0 and countLineBreak in most cases. Since it's for the HTML page we need an array of items (I think) :D 

==> add comment

Reply:
There are two options how we can do this: here in the .ts file with creating an array from the number of elements or we can just send the number of iterations here and use [].constructor(countLineBreaks) as an array to loop.
I think keeping the html more clean is a better way.

I will add a comment here, the fill(0) part is not needed yes, only the creation of the array

Reply:
Comment added and fill(0) removed :)